src/plugins/cosmos/components/modals/Staking/views/GetStarted/GetStartedManager.tsx

```diff
@@ -0,0 +1,35 @@
+import { MemoryRouter, Route, Switch, useHistory } from 'react-router-dom'
+import { useModal } from 'hooks/useModal/useModal'
+
+import { GetStarted } from './GetStarted'
+import {
+  GetStartedManagerProps,
+  GetStartedManagerRoutes,
+  GetStartedRouterProps,
+} from './GetStartedCommon'
+import { LearnMore } from './LearnMore'
+
+export const entries = [GetStartedManagerRoutes.GetStarted, GetStartedManagerRoutes.LearnMore]
+
+const GetStartedRouter = ({ assetId, onClose, history }: GetStartedRouterProps) => (
+  <Switch>
+    <Route path={GetStartedManagerRoutes.GetStarted}>
+      <GetStarted assetId={assetId} history={history} />
+    </Route>
+    <Route path={GetStartedManagerRoutes.LearnMore}>
+      <LearnMore assetId={assetId} onClose={onClose} />
+    </Route>
+  </Switch>
+)
+
+export const GetStartedManager = ({ assetId }: GetStartedManagerProps) => {
+  const { cosmosStaking } = useModal()
+  const history = useHistory()
+  const { close } = cosmosStaking
+
+  return (
+    <MemoryRouter initialEntries={entries}>
+      <GetStartedRouter assetId={assetId} onClose={close} history={history} />
```

Comment:
@gomesalexandre I'm not a big fan about passing the parent history as a prop through 2 components there, if you have an idea about another solution, would be cool!

Reply:
To be able to use a parent's router inside another `<MemoryRouter />, it'll have to be passed down, there's no way around that. The new naming looks much better now, I'd probably go even more declarative and name it `stakingRouterHistory` to make it clear which router that is.